shopinvader/tests/test_settings.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2021 Akretion (http://www.akretion.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from .common import CommonCase
+
+EXPECTED_GET_COUNTRY = (
+    "countries",
+    [
+        "Belgium",
+        "France",
+        "Italy",
+        "Luxembourg",
+        "Spain",
+    ],
+)
+EXPECTED_GET_TITLE = (
+    "titles",
+    [
+        "Doctor",
+        "Madam",
+        "Miss",
+        "Mister",
+        "Professor",
+    ],
+)
+EXPECTED_GET_INDUSTRY = (
+    "industries",
+    [
+        "Administrative",
+        "Agriculture",
+        "Construction",
+        "Education",
+        "Energy supply",
+        "Entertainment",
+        "Extraterritorial",
+        "Finance/Insurance",
+        "Food",
+        "Health/Social",
+        "Households",
+        "IT/Communication",
+        "Manufacturing",
+        "Mining",
+        "Other Services",
+        "Public Administration",
+        "Real Estate",
+        "Scientific",
+        "Transportation",
+        "Water supply",
+        "Wholesale/Retail",
+    ],
+)
+EXPECTED_GET_CURRENCY = ("currencies", ["EUR", "USD"])
+EXPECTED_GET_LANG = ("languages", ["English (US)"])
+
+
+class CommonSettingsCase(CommonCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.backend_1 = cls.env.ref("shopinvader.backend_1")
+        cls.partner = cls.env.ref("shopinvader.partner_1")
```

Comment:
cls.partner is never used. Moreover, do we need a CommonSettingsCase. IMO we don't need it.